conversion/data_from_database.go

```diff
@@ -172,7 +220,8 @@ func (dd *DataFromDatabaseImpl) dataFromDatabaseForDataflowMigration(targetProfi
 		}
 		return common.TaskResult[*profiles.DataShard]{Result: p, Err: err}
 	}
-	_, err = common.RunParallelTasks(sourceProfile.Config.ShardConfigurationDataflow.DataShards, 20, asyncProcessShards, true)
+	r := common.RunParallelTasksImpl[*profiles.DataShard, *profiles.DataShard]{}
+	_, err = r.RunParallelTasks(sourceProfile.Config.ShardConfigurationDataflow.DataShards, 20, asyncProcessShards, true)
```

Comment:
The parallelism should be defined as a static constant

Reply:
So I checked, complex types can't be constant in golang. This isn't possible